sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -67,24 +66,27 @@ def process(self, element):
 
 def run(argv=None):
   """Main entry point; defines and runs the wordcount pipeline."""
-  parser = argparse.ArgumentParser()
-  parser.add_argument('--input',
-                      dest='input',
-                      default='gs://dataflow-samples/shakespeare/kinglear.txt',
-                      help='Input file to process.')
-  parser.add_argument('--output',
-                      dest='output',
-                      required=True,
-                      help='Output file to write results to.')
-  known_args, pipeline_args = parser.parse_known_args(argv)
+  class UserOptions(PipelineOptions):
```

Comment:
We used to have a problem with defining options here (See: https://github.com/apache/beam/pull/1889#issuecomment-276743422)

Other tests started to fail because newly defined options stay in scope affected other tests. Is it resolved? Or this is not happening any more because the options are defaulting to ValueProviders?